engineapi/instance_manager.go

```diff
@@ -16,11 +16,11 @@ import (
 )
 
 const (
-	CurrentInstanceManagerAPIVersion = 2
+	CurrentInstanceManagerAPIVersion = 3
```

Comment:
Don't need to change this version because the process gRPC doesn't change?

Reply:
Yes, should no change `CurrentInstanceManagerAPIVersion` here.